tools/rebuild-web

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Copyright: Ankitects Pty Ltd and contributors
+# License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+# Manually trigger a rebuild and reload of Anki's web stack
+
+set -e
+
+PARENT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+ROOT_DIR=$( cd -- "$( dirname -- "${PARENT_DIR}" )" &> /dev/null && pwd )
```

Comment:
Could this (and the similar line in the other file) be done more elegantly with the following?

```
PARENT_DIR=$(dirname $0)/..
```

Reply:
It does look ugly, doesn't it? First time using this form actually, `dirlink` is my usual choice, typically combined with `readlink` to avoid symlink headaches. But I ran into some `coreutils` differences between Mac and Linux recently, so I wanted to see what SO thinks is the best / most universal practice, and that [lead me](https://stackoverflow.com/a/246128/1708932) to the snippet here.

But arguably this might be overly defensive for our purposes. Either way is fine by me, happy to change it if you want

Reply:
We're not using symlinks here, so I'd favor the more readable version if we don't need the more elaborate one.

Also, aren't parent_dir and root_dir pointing to the same place?

Reply:
`parent` would be pointing to `./tools`, and root to `.` 

But revisiting this again, I think we can just drop the relative paths here completely. I think my original intent here was to make `rebuild-web` callable both from the project root and when `cd`ed into tools (or from any where else, for that matter). I'm not sure if this ever worked, TBH, but it might have because specifying the target with `bazel build "//qt/aqt/data/web"` might have been absolute.

Either way, all of this no longer makes sense as `ninja` expects to be run from the project root, so I switched back to just specifying the paths absolutely, which is also consistent with other scripts under `tools`.